tools/buddy-opt/CMakeLists.txt

```diff
@@ -26,9 +26,11 @@ target_link_libraries(buddy-opt
   LowerRVVPass
   MatMulOptimization
   BatchMatMulOptimization
+  BatchMatMulTileOptimization
```

Comment:
Can we unify `BatchMatMulOptimization` and `BatchMatMulTileOptimization`?

Reply:
There are two technical approaches to achieve tiling in batch matmul. One is similar to BatchMatMulTileOptimization, which uses subviews for tiling. The other is to adapt scf.forall (as in BatchMatMulSCFOptimization) through transformations in MLIR. This is subject to further discussion.